.github/workflows/os_comp.yml

```diff
@@ -25,7 +25,6 @@ jobs:
     - name: Run tests
       run: LD_LIBRARY_PATH=/usr/local/share/boost/1.69.0/lib/:$(rustc --print sysroot)/lib:$LD_LIBRARY_PATH python3 run_tests.py
       env:
-        CI: '1'
```

Comment:
Why remove this? There is still special logic in (at least) `run_project_tests.py` for this env var.

Reply:
Ah, sorry ignore this. I should have read your initial comment more carefully...

Reply:
Yes I brought Actions to Meson early; it took several months for some ordinary features to get added to Actions.